experimental/web/index.html

```diff
@@ -46,14 +48,30 @@
               await this.instantiate();
             },
             instantiate: async function () {
-              const statusOk = 1;
+              // Load proto files from GitHub. We cannot refer to local files unless they are served
+              // via a web server first.
+              // This means that if protobuf definitions are modified locally, this script will not
+              // be able to use the updated definitions.
+              const root = await protobuf.load([
+                'https://raw.githubusercontent.com/project-oak/oak/main/oak_abi/proto/application.proto',
+                'https://raw.githubusercontent.com/project-oak/oak/main/oak_abi/proto/label.proto',
+                'https://raw.githubusercontent.com/project-oak/oak/main/oak_abi/proto/oak_abi.proto',
+              ]);
```

Comment:
Hmmm I see why you went that route, but it's worth mentioning that the overhead of avoiding a web server quickly starts outweighing the overhead of a web server. 

In this case adding one would be simple, we can use npx to load an webserver on demand & use to serve the file. 

A bash file to serve this would look like so
```sh
#!/bin/bash
# Navigate to the project root
cd ../../
# Serve the web emulator
npx servor ./ ./experimental/web/index.html
```

Reply:
Indeed, or maybe I could convince someone (:eyes:) to rewrite this stuff properly at some point...
For now I'll leave it out of this PR anyways, but let's chat about it offline.

Reply:
SGTM